client/src/components/Logo/Logo.tsx

```diff
@@ -0,0 +1,17 @@
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import PetsIcon from '@material-ui/icons/Pets';
+
+const Logo = (): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <Box className={classes.logoBox}>
+      <PetsIcon className={classes.logoIcon} />
+      <Typography className={classes.logoText}>LovingSitter.</Typography>
+    </Box>
```

Comment:
We might not even need to do a Box and styles here. Refer to [Grid](https://material-ui.com/components/grid/)

Reply:
I have swapped this to Grid, and refactored some of the other styling to make it fit